src/QueryEditor.tsx

```diff
@@ -58,15 +71,14 @@ export function QueryEditor(props: Props) {
           value={selectedOption}
         />
       </div>
-      <div className="gf-form">
-        <TextArea
-          style={{ height: 100 }}
-          role="query-editor-input"
-          value={rawQueryText}
-          onBlur={() => props.onRunQuery()}
-          onChange={onQueryTextChange}
-        />
-      </div>
+      <CodeEditor
+        height={calculcateHeight(rawQueryText)}
```

Comment:
I think this should be a good enough reason not to worry about the lack of resize.

Also, seems to be a minor typo here in `calculcateHeight`.